commerce/domain/orchestrators/classes/CartCalculateSample.cls

```diff
@@ -34,44 +34,43 @@ global class CartCalculateSample extends CartExtension.CartCalculate {
 
         // OptionalBuyerActionDetails can be used to optimize the various calculators that are invoked
         CartExtension.CartCalculateCalculatorRequest calculatorRequest = new CartExtension.CartCalculateCalculatorRequest(cart, request.getOptionalBuyerActionDetails());
-        CartExtension.CartValidationOutputList cvoList = cart.getCartValidationOutputs();
 
         if (runPricing) {
             prices(calculatorRequest);
 
-            if (hasErrorLevelCartValidationOutput(cvoList, CartExtension.CartValidationOutputTypeEnum.PRICING)) {
+            if (hasErrorLevelCartValidationOutput(cart.getCartValidationOutputs(), CartExtension.CartValidationOutputTypeEnum.PRICING)) {
```

Comment:
Since `cartValidationOutputs` is memoized in `Cart.apex`, do we still need to remove the `null` check in `Cart.apex` within the Core repo?

Reply:
That would be internal code change. I'm working on that separately.

Reply:
Ok cool just making sure we have that on our radar!